package.json

```diff
@@ -21,19 +21,14 @@
     "debug": "^2.2.0",
     "ibm_db": "^1.0.0",
     "loopback-connector": "^2.3.0",
-    "path": "^0.12.7",
```

Comment:
path is already being used here: https://github.com/strongloop/loopback-ibmdb/blob/0612370804879a879e3593c718ceb3c0289bbe74/lib/globalize.js#L6